twinTrim/flags.py

```diff
@@ -24,10 +16,9 @@
 @click.option("--exclude", multiple=True, help="Files to exclude by name.")
 @click.option("--label-color", default="yellow", type=str, help="Color of the label of progress bar.")
 @click.option("--bar-color", default='#aaaaaa', type=str, help="Color of the progress bar.")
-@click.option("--dry-run", is_flag=True, help="Simulate the process without deleting files.")
```

Comment:
Why did you remove this line? Do not remove anything without any reason, and if you do remove soemthing, always add a comment the reason.